docs/docs/07-forms.md

```diff
@@ -73,7 +80,7 @@ Be aware that, in an attempt to normalize change handling for checkbox and radio
 
 ## Uncontrolled Components
 
-An `<input>` that does not supply a `value` (or sets it to `null`) is an *uncontrolled* component. In an uncontrolled `<input>`, the value of the rendered element will reflect the user's input. For example:
+An `<input>` without a `value` property (or `value` set to `null`) is an *uncontrolled* component:
```

Comment:
`null` is no longer a legal value for uncontrolled inputs.  Setting value to `null` will emit a warning.